CHANGES-5.0.md

```diff
@@ -21,11 +27,13 @@
 
 ## Enhancements
 
+* Add `bootstrap_users_file` configuration to add default Dashboard username list, which is only added when EMQX is first started.
```

Comment:
it was added as `bootstrap_users` then changed to `bootstrap_users_file` in the current release.

Reply:
It's on v5.0.5, https://github.com/emqx/emqx/compare/v5.0.4...v5.0.5